src/service_aggregator.py

```diff
@@ -208,10 +217,11 @@ async def entry(message, guid, coalesce_type, caller) -> (dict, int):
     # return the workflow def so that the caller can see what we did
     final_answer["workflow"] = workflow_def
 
-    if infer:
-        results_cache.set_result(input_id, predicate, qualifiers, source_input, caller, workflow_def, final_answer)
-    elif {"id": "lookup"} in workflow_def or override_cache:
-        results_cache.set_lookup_result(workflow_def, query_graph, final_answer)
+    if overwrite_cache:
```

Comment:
I think this should instead be `if not bypass_cache:` because we want to save everything to cache if it wasn't in there, unless users specifically say they want to bypass the whole thing.